src/test/java/com/Project/accountManager/services/AccountServiceTest.java

```diff
@@ -0,0 +1,152 @@
+package com.Project.accountManager.services;
+
+import com.Project.accountManager.entities.Account;
+import com.Project.accountManager.entities.User;
+import com.Project.accountManager.repository.AccountRepository;
+import com.Project.accountManager.request.AccountCreateRequest;
+import com.Project.accountManager.request.AccountDepositRequest;
+import com.Project.accountManager.request.AccountWithdrawalRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class AccountServiceTest {
+
+    @Mock
+    private AccountRepository accountRepository;
+
+    @Mock
+    private UserService userService;
+
+    @InjectMocks
+    private AccountService accountService;
+
+    @BeforeEach
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testSaveOneAccount() {
+
+        Long userId = 1L;
+        AccountCreateRequest accountCreateRequest = new AccountCreateRequest();
```

Comment:
in such a case it is nice to use `var` .
I highly recommend you to learn when should use `var`
https://www.youtube.com/watch?v=786iemaCJHU